--- libmatchbox-1.9/tests/Makefile.am.ark	2012-01-31 02:20:54.330950356 +0100
+++ libmatchbox-1.9/tests/Makefile.am	2012-01-31 02:21:00.690950243 +0100
@@ -3,7 +3,7 @@ SUBDIRS = menu
 TESTS = dot-desktop pixbuf
 
 AM_CPPFLAGS = @CHECK_CFLAGS@ -I../libmb @GCC_WARNINGS@ @XLIBS_CFLAGS@ @PANGO_CFLAGS@ @PNG_CFLAGS@
-AM_LDFLAGS = -L../libmb -lmb
+AM_LDFLAGS = -L../libmb -lmb -lX11
 
 if HAVE_CHECK
 
--- libmatchbox-1.9/tests/Makefile.in.ark	2012-01-31 02:26:38.128944265 +0100
+++ libmatchbox-1.9/tests/Makefile.in	2012-01-31 02:26:45.327944137 +0100
@@ -219,7 +219,7 @@ target_alias = @target_alias@
 SUBDIRS = menu
 TESTS = dot-desktop pixbuf
 AM_CPPFLAGS = @CHECK_CFLAGS@ -I../libmb @GCC_WARNINGS@ @XLIBS_CFLAGS@ @PANGO_CFLAGS@ @PNG_CFLAGS@
-AM_LDFLAGS = -L../libmb -lmb
+AM_LDFLAGS = -L../libmb -lmb -lX11
 @HAVE_CHECK_TRUE@dot_desktop_SOURCES = dot-desktop.c
 @HAVE_CHECK_TRUE@dot_desktop_LDADD = @CHECK_LIBS@
 @HAVE_CHECK_TRUE@pixbuf_SOURCES = pixbuf.c oh.h
